Runtime/Scripts/VehicleComponents/Actuators/Propeller.cs

```diff
@@ -27,11 +27,9 @@ public class Propeller: LinkAttachment, IROSPublishable
         public bool TorqueUp = false;
         public double DefaultHoverRPM;
 
-        //[SerializeField] private ArticulationBody baseLinkArticulationBody;
-        [SerializeField] private MixedBody baseLinkMixedBody;  // Use MixedBody instead of ArticulationBody
-
-        private float c_tau_f = 8.004e-4f;
-        
+        [SerializeField] private ArticulationBody baseLinkArticulationBody;
```

Comment:
Take the baseLink rigidbody like this, and give them both to the MixedBody in start, instead of "searching up" for a "base_link". Explicit is life :)